tap_pendo/streams.py

```diff
@@ -393,7 +393,12 @@ def sync_substream(self, state, parent, sub_stream, parent_response):
                 if e.get(parent.key_properties[0]) == last_processed:
                     LOGGER.info("Resuming %s sync with %s", sub_stream.name, e.get(parent.key_properties[0]))
                     if isinstance(parent_response, list):
-                        parent_response = parent_response[i:]
+                        try:
+                            starting_index = parent_response.index(e)
```

Comment:
Why not just declare `i` outside of the for loop and increment it like it was an `enumerate`?